### DISCUSSION TITLE
System Tray with obs

### DISCUSSION THREAD

**Discussion by CalfMoon** [hyprwm/Hyprland#5229] [Q&A]
So I read in #2036 what killactive does is asked the program to close and the thing is, when obs is asked to close (press the red button in de) it closes, but there is a option to minimize it to tray by pressing the minimize button. Is there such command for hyprland? or is there a patch or plugin for obs that makes it so that when you ask it to close it minimizes to system tray.
What do you use to hide obs?

**Comment by Gliczy** [comment]
I just click the obs icon in my tray.

**Reply by CalfMoon** [reply]
i don't really find that convenient but it would be more convenient if there was a way to open a program from tray using just the keyboard.
